cluster/charts/crossplane/templates/deployment.yaml

```diff
@@ -92,6 +92,10 @@ spec:
           - name: "WEBHOOK_SERVICE_PORT"
             value: "9443"
           {{- end }}
+          {{- if has "--enable-external-secret-stores" .Values.args }}
```

Comment:
We may want to define a new variable in `_helpers.tpl` as a boolean like `essEnabled` and use it instead if checking the args array everywhere.